modules/custom/az_core/css/az-bef-sidebar.css

```diff
@@ -0,0 +1,90 @@
+.az-bef-vertical ul {
+  list-style: none;
+}
+
+.az-bef-vertical {
+  margin-bottom: 1rem;
+
+  >.form-row>fieldset {
+    margin-bottom: 0;
+    width: 100%;
+
+    >label {
+      font-weight: 600;
+      color: #1e5288;
+      font-size: 1.25rem;
+      line-height: 1.5rem;
+      padding-top: 1rem;
+      border-top: 1px solid #e9ecef;
+      width: 100%;
+    }
+  }
+}
+
+div.checkbox>div.bef-checkboxes>ul {
+  padding: 0;
+  margin: 0;
+
+  >li>div.checkbox>label {
+    color: #1E5288;
+    font-size: 1rem;
+    font-weight: 600;
+  }
+
+  >li>ul {
+    padding-left: 3px;
+    margin: 0;
+
+    >li>ul {
+      padding-left: 10px;
+
+      >li>ul,
+      >li>ul>li>ul,
+      >li>ul>li>ul>li>ul,
+      >li>ul>li>ul>li>ul>li>ul {
+        padding-left: 20px;
+      }
+    }
+  }
+
+  >li:has(ul) {
```

Comment:
[The :has() pseudo-class is not supported by default on Firefox](https://caniuse.com/css-has): users have to enable a specific flag using about:config.

Reply:
Do you have a recommendation of how to remedy this?

Reply:
I think you'll need to solve this with your template, adding a has-submenu class to any li with a ul in it.

Reply:
Could customize this template to add classes. https://git.drupalcode.org/project/better_exposed_filters/-/blob/6.0.x/templates/bef-nested-elements.html.twig?ref_type=heads